### ISSUE TITLE
After upgrade to jsdoc2md 5.x get error: Cannot find module `fs-then-native`

### ISSUE THREAD

**Issue by silverbucket** [jsdoc2md/jsdoc-to-markdown#206]
I was previously on `^3.0.0` and everything worked fine, just updated to `^5.0.3` and now get the following error. 

```
$ jsdoc2md index.js  > API.md
internal/modules/cjs/loader.js:628
    throw err;
    ^

Error: Cannot find module 'fs-then-native'
Require stack:
- /Users/njenning/code/projects/sockethub/packages/sockethub-platform-feeds/node_modules/cache-point/index.js
- /Users/njenning/code/projects/sockethub/packages/sockethub-platform-feeds/node_modules/jsdoc-api/index.js
- /Users/njenning/code/projects/sockethub/packages/sockethub-platform-feeds/node_modules/jsdoc-to-markdown/index.js
- /Users/njenning/code/projects/sockethub/packages/sockethub-platform-feeds/node_modules/jsdoc-to-markdown/bin/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:683:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/Users/njenning/code/projects/sockethub/packages/sockethub-platform-feeds/node_modules/cache-point/index.js:3:12)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:683:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/njenning/code/projects/sockethub/packages/sockethub-platform-feeds/node_modules/cache-point/index.js',
    '/Users/njenning/code/projects/sockethub/packages/sockethub-platform-feeds/node_modules/jsdoc-api/index.js',
    '/Users/njenning/code/projects/sockethub/packages/sockethub-platform-feeds/node_modules/jsdoc-to-markdown/index.js',
    '/Users/njenning/code/projects/sockethub/packages/sockethub-platform-feeds/node_modules/jsdoc-to-markdown/bin/cli.js'
  ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

**Comment by 75lb** [comment]
v5.0.3 works fine for me on macOS.. Try completely uninstalling then re-installing.. failing that, try installing with npm instead of yarn..

**Comment by silverbucket** [comment]
@75lb yeah there appears to be 8 differences in either package versions or entirely missing packages in the `yarn install` versus the `npm install`. Any idea why it effects this package?

**Comment by 75lb** [comment]
According to your error stack, the file `cache-point/index.js` fails to load `fs-then-native`. I checked this project and the required module [is listed in the package.json](https://github.com/75lb/cache-point/blob/master/package.json#L28) so this should work. 

I also installed jsdoc2md using the following command, which worked correctly:

```
$ yarn global add jsdoc-to-markdown
```

I'm sorry, I don't know what is causing your issue. It is not an issue with jsdoc-to-markdown itself, so closing.
